stock/app.py

```diff
@@ -72,10 +100,13 @@ def create_item(price: int):
 
     # Create unique item id
     new_item_id = f"item:{random.getrandbits(ID_BYTES_SIZE)}"
+    db = get_db(new_item_id)
     while db.hget(new_item_id, "item_id"):
         new_item_id = f"item:{random.getrandbits(ID_BYTES_SIZE)}"
+        db = get_db(new_item_id)
 
     # Create new item
+    db = get_db(new_item_id)
```

Comment:
I think this getter here is unnecessary since you already perform it on the initial ```item_id``` generation and then in case the ```item_id``` exists in that db, you repeat the procedure.